### PR TITLE
FIX: Multiple minor UI glitches.

### PR DESCRIPTION
**Pull request by Rene-Damm**
- Fixes "Listen" button not being wide enough for its text on 2019.3.
- Fixes multiple items in the "Layouts" branch of the input debugger receiving the same ID.
- Fixes composites leading to duplicate mentions of controls in the input debugger.

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Due to package verification, the latest version below is the unpublished version and the date is meaningless.
 however, it has to be formatted properly to pass verification tests.
 
+## [1.0.0-preview2] - 2019-?-??
```
Comment (jechter) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Is `preview` followed by `preview2` or `preview1`? (I have been using `preview1` in my currently open PRs). Also, you will need to update your package.json to get tests green likely.

Comment (jechter) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Hmm, if the `com.unity.package-manager-doctools` package we are using uses the scheme correctly, then we are both doing it wrong, and it should be: `1.0.0-preview.1` (with a dot)

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### Fixed
+
+- Selecting a layout in the input debugger no longer selects its first child item, too.
+
+#### Action
+
+- The text on the "Listen" button is no longer clipped off on 2019.3.
+- Controls bound to actions through composites no longer as duplicates in the input debugger.
```
Comment (jechter) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: This sentence a verb.

**Review by jechter — APPROVED** [review]
Some comments, but code changes look fine.

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 1 above)
Comment (jechter) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: I will fix it correctly (using -preview.1) in https://github.com/Unity-Technologies/InputSystem/pull/885, then we can land that and merge it to the other PRs

Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: >I will fix it correctly (using -preview.1) in #885, then we can land that and merge it to the other PRs

Awesome.

>Hmm, if the com.unity.package-manager-doctools package we are using uses the scheme correctly, then we are both doing it wrong, and it should be: 1.0.0-preview.1 (with a dot)

Looking at the semver spec, looks like `preview.1` is indeed the correct way.

>Is preview followed by preview2 or preview1?

Good question. Given that everything else in the version is zero-based, should probably be 1 and not 2.

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 2 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: :)

**Review by douglas-piconi — APPROVED** [review]

**Pull request merged**